en/phones.ftl

```diff
@@ -133,3 +133,66 @@ phone-banner-cta-user = Upgrade now
+
+phone-masking-faq-question-call-length = How long can I talk when I get a call?
+phone-masking-faq-answer-call-length = Each month you get 50 minutes of talking. Once these minutes are used up, you won’t be able to receive forwarded calls until the next month on your billing cycle.
+
+phone-masking-faq-question-can-i-call = Can I call someone with my phone mask?
+phone-masking-faq-answer-can-i-call = No, you can only pick up a forwarded call.
+
+phone-masking-faq-question-can-i-see = Can I see who texted or called me?
+phone-masking-faq-answer-can-i-see = Yes, you can see the number that texted or called you. You can also disable the storage of these records, but you will lose the ability to reply to or block individual callers & texters.
+
+phone-masking-faq-question-can-i-block = Can I block a call or text?
+phone-masking-faq-answer-can-i-block = You can block all forwarding from a single number.
+
+phone-masking-faq-question-spam = What if my phone mask starts getting spam?
+phone-masking-faq-answer-spam = If you start getting spam, you can block the numbers sending you spam.
+
+phone-masking-faq-question-disable-logging = Can I disable the logging of callers or text senders?
+phone-masking-faq-answer-disable-logging = Yes, you can disable logging of numbers from the { -brand-name-relay } dashboard. However, you will no longer be able to reply to texts or block specific numbers, because the log is how we are able to track who sent you a text message.
+
+phone-masking-faq-question-can-i-share = Can I share the number that forwards me text messages?
+phone-masking-faq-answer-can-i-share = If you share this number, nothing will happen—this number is not your phone number mask. It is just the contact number from which { -brand-name-relay } will forward your texts and calls.
+
+phone-masking-faq-question-how-i-save-card = How do I save the { -brand-name-relay } contact card?
+phone-masking-faq-answer-how-i-save-card = Once you upgrade to { -brand-name-relay } phone number masking, we will text you a contact card that contains the number from which you will receive forwarded calls and texts, similar to any contact card that stores the phone number of people who contact you. On most devices, you can select that contact card and save it like any other contact on your phone.
+
+phone-masking-faq-question-install-app = Do I need to install an app to use { -brand-name-relay } phone masking?
+phone-masking-faq-answer-install-app = No, { -brand-name-relay } phone masking works using your device’s standard text messaging and calling apps.
+#   $url (url) - link to Firefox Relay's Privacy Policy, i.e. https://www.mozilla.org/privacy/firefox-relay/
```

Comment:
Should these two comments go above the `answer` string, so the localizers can see it. Right now, they are viewable along with the `question` string.

Reply:
Ah yes: aee42ad